client/packages/common/src/intl/number/IntlNumber.ts

```diff
@@ -2,6 +2,9 @@ import { RegexUtils } from '../../utils/regex';
 import { useCurrency } from '../currency';
 import { SupportedLocales, useIntlUtils } from '../utils';
 
+// According to MDN, which says: 'Possible values are from 0 to 20'
+const MAX_MAXIMUM_FRACTION_DIGITS = 20;
```

Comment:
This issue I came across when trying to run electron:

<img width="1207" alt="Screenshot 2024-03-19 at 3 14 31 PM" src="https://github.com/msupply-foundation/open-msupply/assets/9192912/c0ebbc0b-13f1-474d-a986-f31867a4e19d">

Checking the documentation from mdn, the limit is 20 not 21